project/src/main/kotlin/org/ionproject/core/common/interceptors/ControlAccessInterceptor.kt

```diff
@@ -64,10 +82,10 @@ class ControlAccessInterceptor : HandlerInterceptorAdapter() {
         val requestDescriptor = Request(request.method, request.requestURI, resourceIdentifier)
 
         //Verify the policies, if any fails the next interceptor won't be called and the processing will end
-        pdp.evaluateRequest(tokenHash, requestDescriptor)
+        val token = pdp.evaluateRequest(tokenHash, requestDescriptor)
 
         //Optimization to avoid querying the db again for this same information
-        request.setAttribute("tokenHash", tokenHash)
+        request.setAttribute("token", token)
```

Comment:
Optimization to avoid having to redo database requests to obtain information already provided on authentication...